cmd/ubuntu-image/main.go

```diff
@@ -1,3 +1,8 @@
+// Package ubuntu-image provides a tool used for generating bootable images.
+// You can use it to build two types of Ubuntu images:
+//
+// * Snap-based Ubuntu Core images from model assertions
+// * Classical preinstalled Ubuntu images using image definitions
```

Comment:
Having this package visible on pkg.go.dev is more or less an inconvenient since we do not really want users trying to use ubuntu-image as a library in a go project (I think). 

Maybe we should use this place to also make this very clear and discourage users from doing so?

This https://pkg.go.dev/github.com/canonical/ubuntu-image is already visible but since this is not a package/module per se, it does not appear when searching for "ubuntu-image" in the search bar. So maybe we could "redirect" the user their to get a full presentation of the tool?

Reply:
So, something like:

```go
// Package ubuntu-image provides a tool used for generating bootable images.
// You can use it to build two types of Ubuntu images:
//
// * Snap-based Ubuntu Core images from model assertions
// * Classical preinstalled Ubuntu images using image definitions
//
// ubuntu-image is intended to be used a Snap package available from
// https://snapcraft.io/ubuntu-image
//
// See the project README for more details:
// https://pkg.go.dev/github.com/canonical/ubuntu-image
package main
```
(Or perhaps pointing directly to the repo for the README?)

Reply:
Minor typo:

> ubuntu-image is intended to be used a Snap package

ubuntu-image is intended to be used **as** a Snap package

Otherwise it looks good. I have no strong feeling about the link except that pointing to GH has the advantage of making it easier for the user to dig into the code.

Reply:
PR updated. I opted to not link directly to the `README.md` file because it doesn't seem to make sense. The README is displayed automatically when navigating to the main page of the repo.